packages/mobile/src/stableToken/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { RootState } from 'src/redux/reducers'
```

Comment:
We have an existing a dollar balance selector here: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/stableToken/reducer.ts#L34
Can you use this function or deprecate it?